render-utils.js

```diff
@@ -0,0 +1,24 @@
+export function renderBooks(book) {
+    const bookContainer = document.createElement('a');
+    const bookInfo = document.createElement('p');
+
+    bookContainer.classList.add('book-style');
+
+    bookInfo.textContent = book.book_title;
+    bookContainer.href = `detail/?id=${book.genre_id}`;
+
+    bookContainer.append(bookInfo);
+
+    return bookContainer;
+}
+
+export function renderBook(book) {
+    const bookDetailsContainer = document.createElement('div');
+    const bookInfo = document.createElement('p');
+
+    bookInfo.textContent = book.summary;
```

Comment:
can we try to render some more elements here? book_title, publication_year, author, and genre are in your table and you should have access to them.